tokio/src/io/util/async_write_ext.rs

```diff
@@ -20,20 +21,20 @@ use bytes::Buf;
 cfg_io_util! {
     /// Defines numeric writer
     macro_rules! write_impl {
-        (
-            $(
-                $(#[$outer:meta])*
-                fn $name:ident(&mut self, n: $ty:ty) -> $($fut:ident)*;
-            )*
-        ) => {
-            $(
-                $(#[$outer])*
-                fn $name<'a>(&'a mut self, n: $ty) -> $($fut)*<&'a mut Self> where Self: Unpin {
-                    $($fut)*::new(self, n)
-                }
-            )*
+            (
```

Comment:
Formatting only change. This is a result of getting rustfmt to actually run on this file.

Reply:
It seems like the entire macro got indented four spaces too far. Can you undo this one?